bin/compile

```diff
@@ -118,7 +118,7 @@ restore_cache() {
   if [ "$cache_status" == "valid" ]; then
     local cache_directories=$(get_cache_directories)
     if [ "$cache_directories" == "" ]; then
-      echo "Loading 3 from cacheDirectories (default):"
+      echo "Loading 2 from cacheDirectories (default):"
```

Comment:
you could replace this with: `$(echo $DEFAULT_CACHE | wc -w)` if you wanted to automate that word count ;)